idemp-rest-test/testing/src/models/valueObject.ts

```diff
@@ -55,6 +56,7 @@ export class ValueObject {
 	documentAction?: string;
 	order?: Order;
 	orderLine?: OrderLine;
+	recieveProduct?: ReceiveProduct;
```

Comment:
You can use the `order` property instead of adding this one.

Reply:
Order does not have vendor, receiveProduct extends order and adds vendor that's why I am using it.

Reply:
Can you cast the saved `receiveProduct` as an `order`, then access the `vendor` property through a cast when you need it?